submissions/opeyemiPeter/database/transportation-company.sql

```diff
@@ -41,7 +41,43 @@ CREATE TABLE accidentCases(
  primary key(id),
  foreign key(passengerId) references registeredPassengers(id)
 );
-
+DROP TABLE accidentcases;
 SELECT * FROM accidentCases;
 
 
+-- Task 3 Transportation Data Query.
+
+# 1. Total number of passengers who survived
+	select count(*) from accidentCases where accidentStatus > 0;
+	-- Answers => 342
+
+# 2. total number of unsurvived passengers
+  SELECT COUNT(*) FROM accidentCases WHERE accidentStatus = 0;
+  -- Answers => 549
+
+# 3. Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+	SELECT fullName, sex from registeredPassengers
+	inner join tripDetails on 
+    registeredpassengers.id = tripDetails.passengerId 
+    where (age  < '27') and (embarkationPoint = 
+    ( "Q%"| "C%"));
+    
+# 4. Number of passengers who embarked at Southampton and survived
+	SELECT count(*) from accidentCases left join tripdetails on accidentcases.id = tripdetails.id where accidentStatus > 0 and embarkationPoint = 'S';
```

Comment:
it supposes to be `accidentcases.passengerId = tripdetails.passengerId`. It is that `passengerId` that exist on both table that makes them have a relationship

Reply:
@adaugochi , i have changed it to the passengerId, thanks

Reply:
@adaugochi but do you know that this wont give 218 as its result but 220590.

![image](https://user-images.githubusercontent.com/63324192/108217248-3a6ede00-70e8-11eb-9ba7-ad9f91e2aecc.png)